packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -267,24 +410,22 @@ export class BlobManager {
                 this.redirectTable.delete(blobId);
                 continue;
             }
-            this.blobIds.delete(blobId);
```

Comment:
Why delete this? We still want to delete the blobs that are unused right?

Reply:
I removed `blobIds` and instead I'm adding storage ID -> storage ID entries to the redirect table and using the values (`.values()`) of the redirect table as the list of attached storage IDs. I haven't looked closely at the GC stuff but I don't think it should need to change too much.

Reply:
So, you are saying that redirectTable has all the ids. The keys are all localIds + all storageIds and values are all storageIds.    In that case, the redirectTable should be used as the source of truth for writing the ids in summary right. However, I see that in `summarize` we write `this.storageIds` if its not empty. What this means is that just deleting ids from redirect table won't be enough here. We will have to delete it from `this.storageIds` as well.

Consider this scenario:
- A blob is uploaded in attached state. It is added to `this.storageIds` as `blob1`. It it also added to redirect table as `blob1` -> `blob1`.
- When this blob is unreferenced and deleted, we remove it from the redirect table.
- When writing blob manager's summary, `blob1` still exists in `this.storageIds` and will be written into summary.

Reply:
`this.storageIds` is a getter, it's sourced from the redirect table